prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -338,23 +338,21 @@ struct efa_rdm_ope *efa_rdm_ep_alloc_txe(struct efa_rdm_ep *efa_rdm_ep,
  */
 void efa_rdm_ep_record_tx_op_submitted(struct efa_rdm_ep *ep, struct efa_rdm_pke *pkt_entry)
 {
-	struct efa_rdm_peer *peer;
 	struct efa_rdm_ope *ope;
 
 	ope = pkt_entry->ope;
 	/*
 	 * peer can be NULL when the pkt_entry is a RMA_CONTEXT_PKT,
 	 * and the RMA is a local read toward the endpoint itself
 	 */
-	peer = efa_rdm_ep_get_peer(ep, pkt_entry->addr);
```

Comment:
Similar comment. I prefer you assign `peer = ope->peer` once to save instructions

Reply:
done